uint8 p_ot
uint8 n_ot
uint8 home
uint8 stop
uint8 pcon
uint8 gain2
uint8 p_cl
uint8 n_cl
uint8 mode
uint8 reserved1
uint8 emg
uint8 a_rst
uint8 sv_on
uint8 spd1_lvsf1
uint8 spd2_lvsf2
uint8 spd3
uint8 start
uint8 pause
uint8 regt
uint8 hstart
uint8 isel0
uint8 isel1
uint8 isel2
uint8 isel3
uint8 isel4
uint8 isel5
uint8 absrq
uint8 jstart
uint8 jdir
uint8 pclear
uint8 aovr
uint8 reserved2